### PR TITLE
remove gitops tracers

### PR DESCRIPTION
**Pull request by salahalsaleh**

### CODE REVIEW

File: pkg/midstream/write.go [hunk 1]
```diff
@@ -102,20 +93,18 @@ func (m *Midstream) mergeKustomization(existing *kustomizetypes.Kustomization) {
 	newResources := findNewStrings(m.Kustomization.Resources, existing.Resources)
 	m.Kustomization.Resources = append(existing.Resources, newResources...)
 
-	m.Kustomization.CommonAnnotations = mergeMaps(m.Kustomization.CommonAnnotations, existing.CommonAnnotations)
```
Comment (divolgin) [inline pkg/midstream/write.go:105]: this is needed

File: pkg/midstream/write.go [hunk 2]
```diff
@@ -102,20 +93,18 @@ func (m *Midstream) mergeKustomization(existing *kustomizetypes.Kustomization) {
...
 }
 
 func mergeMaps(new map[string]string, existing map[string]string) map[string]string {
 	merged := existing
-	if merged == nil {
-		merged = make(map[string]string)
-	}
-	for key, value := range existing {
-		merged[key] = value
+	if new != nil {
```
Comment (divolgin) [inline pkg/midstream/write.go:101]: why this change?  it is safe to use nil map in a loop

File: pkg/midstream/write.go (hunk 1 above)
Comment (salahalsaleh) [reply pkg/midstream/write.go:105]: hmmm, before gitops tracers were added, this statement never existed. why is it needed now?

File: pkg/midstream/write.go (hunk 2 above)
Comment (salahalsaleh) [reply pkg/midstream/write.go:101]: oh, i did not know that, will update 👍
